src/NuGet.Core/NuGet.Protocol/Strings.resx

```diff
@@ -519,8 +519,8 @@ The "ms" should be localized to the abbreviation for milliseconds.</comment>
     <value>Problem starting the plugin '{0}'. {1}</value>
     <comment>0 - plugin path, 1 - error message </comment>
   </data>
-  <data name="PushCommandSkipDuplicateAdvertise" xml:space="preserve">
-    <value>See help for push option to automatically skip duplicates.</value>
+  <data name="PushCommandSkipDuplicateAdvertiseNuGetExe" xml:space="preserve">
+    <value>To skip already published packages, use &lt;command&gt; -skipduplicate</value>
```

Comment:
Isn't the command -SkipDuplicate? 
Use the same casing.

Reply:
It's actually case-insensitive

Reply:
I think the point is that our messages should be consistent. Use the same casing in docs, warning messages, help messages, etc. Using Pascal Casing makes it easier to read than all lowercase.